packages/p2p/src/components/floating-rate/__test__/flotaing-rate.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
This file's name is not `floating` but `FLOTAING`

Reply:
Will change it